drivers/eeprom/eeprom_at2x.c

```diff
@@ -487,12 +487,12 @@ static int eeprom_at25_read(const struct device *dev, off_t offset, void *buf,
 static int eeprom_at25_wren(const struct device *dev)
 {
 	const struct eeprom_at2x_config *config = dev->config;
-	uint8_t cmd = EEPROM_AT25_WREN;
-	const struct spi_buf tx_buf = {
-		.buf = &cmd,
+	static const uint8_t cmd = EEPROM_AT25_WREN;
+	static const struct spi_buf tx_buf = {
```

Comment:
Not sure the `static const` changes are okay. Some SPI-controller drivers may be using DMA yet unable to DMA transfer from flash. I'd prefer to leave these as-is.